databroker/tests/test_broker.py

```diff
@@ -389,6 +389,11 @@ def test_filters(db_empty, RE):
 
     assert header['start']['uid'] == ken_calib_uid
 
+    db.clear_filters()
+    db.add_filter(start_time='2017')
+    db.add_filter(start_time='2017')
```

Comment:
I think you are misunderstanding Tom's question. This test is good as is for testing that duplicate are ignored. He is suggesting that you check that this doesn't make _all_ things ignored, only duplicates.